code/go/internal/validator/common_spec.go

```diff
@@ -19,6 +19,11 @@ type commonSpec struct {
 	DevelopmentFolder  bool             `yaml:"developmentFolder"`
 
 	Limits commonSpecLimits `yaml:",inline"`
+
+	// Release type of the spec: beta, ga.
+	// Packages using beta features won't be able to go GA.
+	// Default release: ga
+	Release string `yaml:"release"`
```

Comment:
Should we check somewhere that we only use `beta` or `ga` here? Or even only `beta`?

Reply:
I added a condition to check for "beta" and "ga" and report an error whenever there is a different level used.